test/packages-src/bonjour/bonjour.bat

```diff
@@ -0,0 +1 @@
+ECHO bonjour!
```

Comment:
Is it worth keeping this in sources as it is also contained by the bonjour-v1.pkg file, and it is not used as a standalone file by tests ?